examples/browser/package.json

```diff
@@ -24,19 +24,23 @@
     "clean": "rimraf lib",
     "prepare": "npm run clean && npm run build",
     "build": "npm run build:localdeps && npm run build:backend && npm run build:frontend",
+    "build:dev": "npm run build:localdeps:dev && npm run build:backend && npm run build:frontend",
```

Comment:
for examples i think it is not important they always can be build in dev mode, we are not going to publish them